hack/windows/Prepare-AntreaAgent.ps1

```diff
@@ -7,7 +7,14 @@ This script prepares environment needed by antrea-agent which includes:
 - Cleaning stale Antrea network resources if they exist.
 - Prepare a network interface which is needed by kube-proxy. Without the interface, kube-proxy cannot
   provide the proxy for Kubernetes Services.
+
+.PARAMETER InstallKubeProxy
+Specifies whether kube-proxy interface is included in the installation. If false, this interface will not
+be installed on the host.
 #>
+Param(
+    [parameter(Mandatory = $false)] [bool] $InstallKubeProxy = $true
```

Comment:
is it expected to make it default to true?

Reply:
Had discussion about it https://github.com/antrea-io/antrea/pull/4980#discussion_r1239131472.